textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/team3/team3lucenequeryexample/LuceneQueryExample.java

```diff
@@ -0,0 +1,94 @@
+package edu.uci.ics.textdb.sandbox.team3.team3lucenequeryexample;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.Tokenizer;
+import org.apache.lucene.analysis.ngram.NGramTokenizer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.FieldType;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.index.IndexOptions;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+
+import edu.uci.ics.textdb.common.constants.LuceneConstants;
+
+/* 
+ * This example shows that lucene can process manually translated binary expressions.   
+ */
+
+public class LuceneQueryExample {
```

Comment:
It may be good to rename it to "LuceneNgramQueryExample".  Not a big deal though.